Tools/AP_Periph/Parameters.cpp

```diff
@@ -209,6 +209,15 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
     // @User: Advanced
     GSCALAR(serial_number, "BRD_SERIAL_NUM", 0),
 
+#ifdef HAL_GPIO_PIN_LED
```

Comment:
this adds a LOCATE param to every peripheral, I think users would find it confusing as to why their GPS has a LOCATE param
please use a DEBUG bit to change the existing LED flashing code to flash at a high rate